_i18n/de/resources/moneropedia/denominations.md

```diff
@@ -1,39 +1,39 @@
-|-----------:|:--------:| -----------------:|
-| piconero   | 10^-12   | 0.000000000001    |
-| nanonero   | 10^-9    | 0.000000001       |
-| micronero  | 10^-6    | 0.000001          |
-| millinero  | 10^-3    | 0.001             |
-| centinero  | 10^-2    | 0.01              |
-| decinero   | 10^-1    | 0.1               |
-|============+==========+===================|
-| **monero** | **10^0** | **1**             |
-|============+==========+===================|
-| decanero   | 10^1     | 10                |
-| hectonero  | 10^2     | 100               |
-| kilonero   | 10^3     | 1,000             |
-| meganero   | 10^6     | 1,000,000         |
-|------------+----------+-------------------|
+|------------+---------------+-------------------|
+| Name       | Dezimalsystem | Betrag            |
+|-----------:|:-------------:| -----------------:|
+| Piconero   | 10^-12        | 0,000000000001    |
+| Nanonero   | 10^-9         | 0,000000001       |
+| Micronero  | 10^-6         | 0,000001          |
+| Millinero  | 10^-3         | 0,001             |
+| Centinero  | 10^-2         | 0,01              |
+| Decinero   | 10^-1         | 0,1               |
+|============+===============+===================|
+| **Monero** | **10^0**      | **1**             |
+|============+===============+===================|
+| Decanero   | 10^1          | 10                |
+| Hectonero  | 10^2          | 100               |
+| Kilonero   | 10^3          | 1.000             |
+| Meganero   | 10^6          | 1.000.000         |
+|------------+---------------+-------------------|
 
-### In-depth Information
+### Ausführliche Informationen
 
-Support for input using SI prefixes was [added to the Monero codebase](https://github.com/monero-project/monero/pull/1826) on March 3, 2017 by [Moneromooo](https://github.com/moneromooo-monero). The smallest unit of Monero (10^-12 XMR) was originally called a tacoshi in honor of user [Tacotime](https://bitcointalk.org/index.php?action=profile;u=19270), an early Monero contributor and was later renamed for ease of use and consistancy.
+Die Unterstützung dessen, bei der Eingabe die SI-Präfixe nutzen zu können, wurde am 3. März 2017 von [Moneromooo](https://github.com/moneromooo-monero) [zu Moneros Codebasis hinzugefügt](https://github.com/monero-project/monero/pull/1826). Zu Ehren des Nutzers [Tacotime](https://bitcointalk.org/index.php?action=profile;u=19270), einem frühen Mitwirkenden Moneros, wurde die kleinste Einheit von Monero (10^-12 XMR) zunächst "Tacoshi" genannt; zugunsten der Nutzerfreundlichkeit und Konsistenz wurde dies allerdings später geändert.
 
-### Monerod Implementation
+### Monerod-Implementierung
 
-The smallest fraction of Monero in the current monerod implementation is also known as the @atomic-unit, which is currently one piconero.
+Der kleinste Bruchteil von Monero in der aktuellen Monerod-Implementierung ist auch bekannt als @Kleinste-Einheit, was derzeit ein Piconero ist.
```

Comment:
It was already translated as "Kleinste Einheit" and I liked it more like that than "Atomare EInheit" for some reason. Somehow it makes "more sense" to me in this context, but I cannot really explain why. Maybe because "atomar" I automatically link to the world of  and it feels weird in a "financial context". In the end, it's both the same, with "atomar" being a bit more technical, or am I missing something?

Reply:
I only flagged this one because they used "atomic-unit" in English too. Not a finance expert so I'm not sure which term would be better here.

Reply:
I was hoping for some other (more technical) input here, but it didn't "arrive". Since I think both versions are fine, it will be "Kleinste Einheit" for now, but this can obviously be changed anytime.